docs/content/docs/_index.md

```diff
@@ -811,7 +811,7 @@ Example: `{{ value | join(sep=" // ") }}`
 If value is the array `['a', 'b', 'c']`, the output will be the string "a // b // c".
 
 #### length
-Returns the length of an array or a string, 0 if the value is not an array.
+Returns the length of an array, an object, or a string. Always 0 for other types.
```

Comment:
Can you change the other types to error instead?

Reply:
Done. The comments were removed, I found no statement about returning ("throwing") an error.